aim/agent/aid/universes/aci/converter.py

```diff
@@ -98,7 +99,8 @@ def fault_inst_to_resource(converted, helper, to_aim=True):
 
 def fv_rs_dom_att_converter(object_dict, otype, helper,
                             source_identity_attributes,
-                            destination_identity_attributes, to_aim=True):
+                            destination_identity_attributes, to_aim=True,
+                            disable_micro_segmentation=False):
```

Comment:
This variable is unused here, as the method is reading directly from the config file.

Reply:
Good catch (this was left over after the initial attempt at passing this in)! Will remove and update the patch.